L1Trigger/Phase2L1ParticleFlow/plugins/L1SeedConePFJetProducer.cc

```diff
@@ -200,12 +228,19 @@ std::vector<l1t::PFJet> L1SeedConePFJetProducer::convertHWToEDM(
     l1t::PFJet edmJet(l1gt::Scales::floatPt(gtJet.v3.pt),
                       l1gt::Scales::floatEta(gtJet.v3.eta),
                       l1gt::Scales::floatPhi(gtJet.v3.phi),
-                      /*mass=*/0.,
+                      0,  // dont set the mass by default
                       gtJet.v3.pt.V,
                       gtJet.v3.eta.V,
                       gtJet.v3.phi.V);
     edmJet.setEncodedJet(l1t::PFJet::HWEncoding::CT, jet.pack());
-    edmJet.setEncodedJet(l1t::PFJet::HWEncoding::GT, jet.toGT().pack());
+
+    if (wideConeJet) {
+      edmJet.setEncodedJet(l1t::PFJet::HWEncoding::GT, jet.toGTWide().pack());
```

Comment:
We should define a new `l1t::PFJet::HWEncoding::GTWide` for this since this introduces a new packed format type.